Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.rst

```diff
@@ -0,0 +1,113 @@
+(currently not implemented), as well as in response to safety interlocks and software-based emergencies. 
+While the primary objective of the emergency e-stop procedure is to provide a quick and reliable means of halting the movement of the telescope and dome to avoid accidents and injuries, 
+it also serves the purpose of safeguarding the system.
+
+Below you will find a brief overview of the emergency stop procedure, covering:
+
+- The location of the relevant switches, 
+- How to activate the emergency stop, and
+- How to determine its current status.
+  
+
+.. _E-Stop-Procedure-Where-to-Find-It:
+
+Where to Find It
+================
+
+The emergency stop system in the Auxiliary Telescope can be triggered from three different locations. 
+One is located in the main ``AT Control Cabinet`` on the first floor. 
+A second one is located in the ``Dome Control Box`` on the second floor. 
+Both of these locations can be used to both engage and disengage the emergency stop.
+
+.. figure:: ./_static/e-stop-dome-control-box.jpeg
+    :name: e-stop-dome-control-box
+
+    Screenshot of the e-stop system located in the Dome Control Box.
+
+Additionally, a third emergency stop button is available upstairs in the control box used for manual movement of the dome components, 
+as illustrated in the accompanying figure. 
+It is worth noting, however, that this control box can only be used to initiate the emergency stop and cannot be used to release it, 
+as it does not have a ``Start`` button on it.
+
+.. figure:: ./_static/e-stop-up-stairs.jpeg
+    :name: e-stop-up-stairs
+
+    Screenshot of the e-stop system located upstairs.
+
+In the future, an emergency stop will also be added to the main telescope control room. 
+For now, the emergency e-stop system available in there will only work for the main telescope components.
+
+.. _E-Stop-Procedure-How-Engage-Disengage-It:
+
+How to Engage/Disengage It
+==========================
+
+**Engaging** the emergency stop system is a straightforward process that is both efficient and reliable. 
+To trigger it, simply **pull the Emergency Stop button**. To **release it**, simply **turn the red button clockwise** a quarter or so
+and then **pull the Start** button.
+
+.. figure:: ./_static/e-stop-buttons.jpeg
+    :name: e-stop-buttons
+
+    Screenshot the e-stop ``Emergency Stop`` and ``Start`` buttons.
+
+**Verifying** that the emergency stop system was properly engaged is also easy and straightforward. 
+You just have to look at the computer located on the first floor of the AuxTel building. 
+If the emergency stop was correctly triggered, you will see the indicator light up red, as depicted in the accompanying figure. 
+Conversely, when the emergency stop is released, the red light will be turned off, indicating that the system has been successfully restarted.
+From that points onwards, the telescope will be released to move, either local or remotely.
```

Comment:
Quick question - "to move either local or remotely", means now the telescope can be operated as usual with LOVE, notebook, EUI, etc? Then, maybe "From that point onwards, the telescope can be operated again" or a similar expression might be clearer (and "local or remotely" would not be necessary).